### PR TITLE
docs: fix grammar in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by fxck**
https://github.com/angular/material2/pull/4230#pullrequestreview-34435184

cc @kara

### CODE REVIEW

**Comment by Toxicable** [comment]
Your commit message does not fit the required format. Please read the guidelines to contributing here https://github.com/angular/angular/blob/master/CONTRIBUTING.md

**Review by wardbell — APPROVED** [review]
LGTM

**Review by wardbell — APPROVED** [review]
LGTM after fixing your commit message

**Review by IgorMinar — APPROVED** [review]

**Comment by matsko** [comment]
@fxck please rebase

**Comment by wardbell** [comment]
I resolved. Also reworked a passive sentence and got rid of "Please", which we're not supposed to say per Google doc guide lines.

**Comment by matsko** [comment]
there's still a rebase issue. And it looks like there's also an issue with circleci

**Comment by fxck** [comment]
Uh, sorry, been busy, do you still need me to do something?

**Comment by matsko** [comment]
@fxck @wardbell Guys please fix the conflicts and squash into one commit.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,15 +17,16 @@ Help us keep Angular open and inclusive. Please read and follow our [Code of Con
 
 ## <a name="question"></a> Got a Question or Problem?
 
-Please, do not open issues for the general support questions as we want to keep GitHub issues for bug reports and feature requests. You've got much better chances of getting your question answered on [Stack Overflow](https://stackoverflow.com/questions/tagged/angular) where the questions should be tagged with tag `angular`.
+Do not open issues for general support questions as we want to keep GitHub issues for bug reports and feature requests. You've got much better chances of getting your question answered on [StackOverflow](https://stackoverflow.com/questions/tagged/angular) where the questions should be tagged with tag `angular`.
```
Comment (gkalpak) [inline CONTRIBUTING.md]: Why `Stack Overflow` --> `StackOverflow`? I thought the official version was as two words.

**Comment by jasonaden** [comment]
@fxck @wardbell I'm removing the "merge" pr action as we need it to pass linting. Looks like it's the commit message, so please squash into one commit with the correct message.

**Comment by petebacondarwin** [comment]
Closing in favour of #17855 as it has been fixed up and has signed CLA.

**Pull request closed without merging**